src/contracts/pods/EigenPod.sol

```diff
@@ -754,7 +753,7 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
             partialWithdrawalAmountGwei
         );
 
-        sumOfPartialWithdrawalsClaimedGwei += partialWithdrawalAmountGwei;
+        sumOfPartialWithdrawalsClaimedViaMerkleProvenGwei += partialWithdrawalAmountGwei;
```

Comment:
this function should be removed, partial withdrawal processing in general outside of zk. if you think otherwise, can you write something for discussion?

Reply:
Sure where do you want me to write this?  I think its pretty simple.  In the event Risc0 goes down, we jjust have to make a very simple transaction (can be done via etherscan) vs an upgrade.  I agree having non functional code in the contract isn't good but having Risc0 go down and being able to react really quick I think is important too

Reply:
I do think leaving this code in reduces the trust in the system, at least in some cases.
I can see the potential for bugs in the interactions between this code and the new zk-batched proving code, but it does seem like we at least have to deal with _most_ of these issues simply due to the upgrade happening at all.